datasets/pk/proscribed_persons/crawler.py

```diff
@@ -0,0 +1,33 @@
+from zavod import Context
+from zavod import helpers as h
+
+
+def crawl_person(context: Context, row: dict):
+    person_name = row.pop("name")
```

Comment:
I noticed some @'s in some names - let's split on @, add the first element to name, and the rest as aliases